db/seeds.rb

```diff
@@ -7,3 +7,7 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+User.create(email: "user@laterolabs.com", password: "password")
+
+User.create(email: "admin@example.com", password: "password")
```

Comment:
These should use `first_or_create` and should also set `username` to avoid uniq index errors